site/src/docs/components/cookie-consent/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+slug: '/components/cookie-consent'
+title: 'CookieConsent'
+navTitle: 'CookieConsent'
+---
+
+import { Link } from 'hds-react';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Usage
+
+This documentation page is about HDS CookieCompliance component. If you are looking for documentation about cookies in general, please refer to <a href="/patterns/cookies">HDS cookie pattern documentation page</a>.
```

Comment:
This link is not working on the demo site. It needs to use the gatsby `withPrefix` function. And also, we should probably use the HDS Link component in here.

Reply:
Fixed